src/main/java/uk/gov/pay/webhooks/webhook/resource/WebhookResponse.java

```diff
@@ -12,13 +14,20 @@
 import java.util.List;
 
 public record WebhookResponse(
+        @Schema(example = "eo29upsdkjlk3jpwjj2dfn12")
         @JsonProperty("service_id") String serviceId,
         @JsonProperty("live") Boolean live,
+        @Schema(example = "https://example.com")
         @JsonProperty(FIELD_CALLBACK_URL) String callbackUrl,
+        @Schema(example = "Webhook description")
         @JsonProperty(FIELD_DESCRIPTION) String description,
+        @Schema(example = "gh0d0923jpsjdf0923jojlsfgkw3seg")
         @JsonProperty("external_id") String externalId,
+        @Schema(example = "ACTIVE")
         @JsonProperty(FIELD_STATUS) WebhookStatus status,
+        @Schema(example = "\"2022-04-05T17:07:15.281Z\"")
```

Comment:
Could we remove the extra quote marks here?

Reply:
This is required for swagger UI to display correctly. It may just be an issue swagger UI. Removed it for now